frontend/src/components/Table/MoneyCell.tsx

```diff
@@ -39,4 +50,41 @@ export const EditableMoneyCell = ({
   );
 };
 
-export const MoneyCell = ({ cell: { value } }: CellProps<IProperty, number>) => formatMoney(value);
+/**
+ * Provides a currency editing input if the user is allowed to edit the property and if the property is land/subdivision.
+ * @param param0 Configuration properties for this editable money cell.
+ */
+export const EditableLandMoneyCell = ({
```

Comment:
This was required to handle when a building row shouldn't allow editing of `Land Assessment`